src/test/ui/hrtb/hrtb-conflate-regions.nll.stderr

```diff
@@ -1,14 +1,20 @@
-error: higher-ranked subtype error
+error: implementation of `Foo` is not general enough
   --> $DIR/hrtb-conflate-regions.rs:27:10
    |
 LL | fn b() { want_foo2::<SomeStruct>(); }
-   |          ^^^^^^^^^^^^^^^^^^^^^^^^^
+   |          ^^^^^^^^^^^^^^^^^^^^^^^^^ implementation of `Foo` is not general enough
+   |
+   = note: `SomeStruct` must implement `Foo<(&'0 isize, &'1 isize)>`, for any two lifetimes `'0` and `'1`...
+   = note: ...but it actually implements `Foo<(&'2 isize, &'2 isize)>`, for some specific lifetime `'2`
```

Comment:
Ideally we would point at the `impl` and the `where` clause in `want_foo2`.

Reply:
In both NLL and non-nll mode, we don't point at the `impl` or `where` clause. I think this could be an separate diagnostics issue, independent of the NLL transition.